src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListArtworksGrid.tsx

```diff
@@ -145,6 +145,9 @@ const ArtworkListArtworksGrid: FC<ArtworkListArtworksGridProps> = ({
 
   return (
     <>
+      {/* Note: this <Jump> element is also used as a sentinel by the
+      shouldScroll() helper in the ArtworkListContent component  */}
+
       <Jump id="artworksGrid" />
```

Comment:
I used this as the element for which to check visibility, because it was already available. 

However we may want to add a new sentinel element instead, a little further down and closer to the artwork grid, if we want to make it a bit more likely that we scroll content into view.